symfony.lock

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
A bundle or a library should not contains this file

Reply:
I couldn't find a way to use flex to force a specific symfony version, without recipe installation.
Without this, during composer install, a [few files are added, and make the CI fail](https://github.com/BedrockStreaming/StaleCacheBundle/runs/8032948453?check_suite_focus=true)...

Contrarily to composer.lock, this file does nothing but prevent adding those files. It does not impact the bundle installation.
I think it's fine then.